server/cluster/unsafe_recovery_controller.go

```diff
@@ -102,6 +102,7 @@ func (u *unsafeRecoveryController) RemoveFailedStores(failedStores map[uint64]st
 		}
 		u.storeReports[s.GetID()] = nil
 	}
+	u.cluster.PauseOrResumeScheduler("all", 3600)
```

Comment:
1h maybe too long, how about 10min?

Reply:
The time can load from request, if not, it  can use default value.

Reply:
How long should the schedulers be paused does not really matter, right? In my opinion, they should be paused "indefinitely" until released by unsafe recovery controller again.

If you guys agree with it, we can achieve this goal by checking `c.UnsafeReocoveryController().IsRunning()` whenever PD wants to schedule somethings, instead of using `PauseOrResumeScheduler()`, like we did to `AskSplit`.

Reply:
In your opinion, the recovery duration should be less than 1h, Is it right.

Reply:
In most cases, it should, but based on the "no assumption" rule, I think it may be better to make it dynamic (suspend all the schedulers until release them explicitly), so it still works in some "extreme cases". I am proposing that, instead of pausing for a fixed amount of time, make it infinite. If everything works fine, this approach may be better. While, the up side of using "pause" is that, if "unsafe recovery" gets stuck, the schedulers won't be blocked forever. However, if something really goes wrong with the unsafe recovery, we need to restart the PD, and, after that, the schedulers will be resumed anyway.

Reply:
please make it configurable, in case there's need to change in a specific scenario.

Reply:
It is now configurable through the optional command line argument.